imageUpload.js

```diff
@@ -129,7 +129,9 @@ function imageProviderFSBackend(storageRootDir) {
           let subdir = req.params.image_id.slice(0, 3),
             fname = req.params.image_id.slice(3);
           const imgPath = path.join(storageRootDir, basePath, subdir, fname);
+          const dirPath = path.join(storageRootDir, basePath, subdir);
           await fs.unlink(imgPath);
+          await fs.remove(dirPath);
```

Comment:
@Ren22 why to remove the whole subdirectory? It might contain other images.

Reply:
Is there a way to have more than one image in the same folder? I just found it to be so that for every image there's only one corresponding folder. But this is easy to change